### ISSUE TITLE
Service Mark special legal character not read by screen reader on any punctuation setting

### ISSUE THREAD

**Issue by levyadams** [nvaccess/nvda#11547]
### Steps to reproduce:

Read the following sentence with the screen reader and notice it does not read the legal special character:

"John's cat didn't sell products, but services, so John's cat used the ℠ next to his advertised services."

### Actual behavior:

℠ is a legally binding special character similar to ™ and ®, however only the latter characters are read. This becomes a 508 issue with businesses who are legally required to be accessible.

### Expected behavior:

The screen reader should read the service mark as, "Servicemark" to the keyboard user as it does with other legal special characters at the verbosity/punctuation level other special legal characters are read out to the user.

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:
2019.3.1
#### Windows version:
Windows 10 Enterprise 10.0.17763
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
n/a
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No, I'm sorry but I don't have bandwidth personally for that.
#### If addons are disabled, is your problem still occuring?
No addons used
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No, this is a fresh install of NVDA screen reader program.

Thank you!

**Comment by lukaszgo1** [comment]
This is a one line fix to source\locale\en\symbols.dic. Should probably be labelled as a quickFix and goodForNewDev

**Comment by levyadams** [comment]
I probably was still somehow to mess this up but lemme know if that works! Thanks!!

**Comment by feerrenrut** [comment]
Fixed with #11559
